plugin.json

```diff
@@ -4,10 +4,11 @@
     "meta": {
         "label": "Google Cloud Translation",
         "category": "Natural Language Processing",
-        "description": "Use Google Cloud basic translation API to perform translation",
+        "description": "Use Google Cloud Translation API to translate text data to another language",
         "author": "Dataiku (Alex COMBESSIE, Arnaud d'Esquerre)",
         "icon": "icon-gcp-translation-api icon-cloud",
         "licenseInfo": "Apache Software License",
+        "supportLevel": "NOT_SUPPORTED",
```

Comment:
Perhaps worth asking to Adrien & Krishna? Since it's good and useful work, may be worth bumping to tier 2?